recipes/giflib/5.2.x/CMakeLists.txt

```diff
@@ -19,8 +19,36 @@ if(NOT BUILD_SHARED_LIBS)
 endif()
 
 add_library(${PROJECT_NAME} ${SOURCE_FILES})
+if(WIN32 AND BUILD_SHARED_LIBS)
+    set_property(TARGET ${PROJECT_NAME} PROPERTY PREFIX "")
+endif()
```

Comment:
to create `gif.dll` for MinGW?

Reply:
Yes, probably unnecessary but that will let mingw and MSVC create the same libraries.

Reply:
Yes, I still don't understand why it's not the default in CMake.